main.go

```diff
@@ -1,10 +1,17 @@
 package main
 
 import (
+	"bufio"
```

Comment:
i guess they deprected `ioutils` after go1.16 or smth so I was using `bufio` and `os` for file io ops but kurtosis uses go1.15 so might need to change my go version and go back to `ioutils`